configs/containers/specs/jedi-ci.yaml

```diff
@@ -15,7 +15,7 @@
     eckit@1.28.3,
     ecmwf-atlas@0.40.0 +fckit +trans +tesselation +fftw,
     fiat@1.4.1,
-    ectrans@1.2.0 +fftw,
+    ectrans@1.2.0,
```

Comment:
this must have slipped somehow in my first round of reviews. Why is this being turned off? Or is the default for variant `fftw` true?

Reply:
The intel oneapi definition requires the ectrans `+mkl ~fftw` variant and the container definition ended up with both variants listed causing the concretizer to fail because it could not unify.

We could potentially add the `+fftw` variants to all of the other containers if this is needed. I noticed that the common config for ectrans also had the fftw variant [disabled recently](https://github.com/JCSDA/spack-stack/commit/491b30fa8922bf2e0bbbb4774227095a6c1bf3b1)

Reply:
I wonder how this works for Intel Classic (%intel)

Reply:
well, the easy answer is that it works because intel classic doesn't specify a variant therefore gets the +fftw variant. I take it that we should be using the `+mkl ~fftw` variant for both intel oneapi and classic?

Reply:
I think that would be better. Although Intel Classic is a dying horse ...

Reply:
I've added the correct variant to the intel classic container. Its building now and I'll close this thread once I verify the change.